Formula/ogre.rb

```diff
@@ -0,0 +1,129 @@
+class Ogre < Formula
+  desc "Scene-oriented, flexible 3D engine"
+  homepage "http://www.ogre3d.org/"
+  url "https://github.com/OGRECave/ogre/archive/v1.10.9.tar.gz"
+  sha256 "85ba2cc6a35c67ff93a9a498af9f8f3113fd3a16e7cc43c18b9769d8bf1e9101"
+
+  option "with-java-component", "Build with java component"
+  option "with-python-component", "Build with python component"
+
+  needs :cxx11
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost" => :optional
+  depends_on "freeimage"
+  depends_on "freetype"
+  depends_on "harfbuzz"
+  depends_on "libpng"
+  depends_on "libzzip"
+  depends_on "openexr"
+  depends_on "sdl"
+  depends_on "sdl2"
+
+  if build.with?("java-component") || build.with?("python-component")
+    depends_on "swig"
+    depends_on :java if build.with? "java-component"
+    depends_on :python if build.with? "python-component"
+  end
+
+  def install
+    if build.with? "boost"
+      unless Tab.for_name("boost").cxx11?
+        odie "Boost build with \"--c++11\" is needed because this formula needs c++11."
```

Comment:
I believe correct way to do this is
```ruby
depends_on "boost" => [:optional, :cxx11]
```

Reply:
This isn't permitted at all actually. It needs to use the standard boost.